Dynamic_Bar.lua

```diff
@@ -19,34 +17,28 @@ Type:UsesAttributes("value, maxValue, valueColor")
 Type:UsesAttributes("state")
 Type:UsesAttributes("texture")
 
-TMW:RegisterUpgrade(45013, {
```

Comment:
Your type didn't exist during version 4.5.0 (which dates to Jul 26, 2011), so this isn't needed. (`TMW:RegisterUpgrade` is for migrating settings from old versions to new versions.)